test/dynamo/test_misc.py

```diff
@@ -3550,6 +3550,29 @@ def inner(y: typing.List[Variable]):
         self.assertEqual(cnts.frame_count, 2)
         self.assertEqual(cnts.op_count, 2)
 
+    def test_function_generic_alias_annotation(self):
```

Comment:
This is just copied from test_function_annotation above, with "typing.List" replaced by "list".